package.json

```diff
@@ -5,7 +5,8 @@
   "workspaces": [
     "configs/*",
     "packages/*",
-    "apps/*"
+    "apps/*",
+    "devkit/e2e"
```

Comment:
is there an issue when using `devkit/*`?

Reply:
Doesn't present any issue and keeps the styles of the other workspaces.
Fixed ✅